students/Chris_Hallen/lesson02/fizzBuzz.py

```diff
@@ -0,0 +1,14 @@
+def fizzBuzz():
+    num = 1
+    while num <= 100:
```

Comment:
note that you can use `range` function which reduces the need of keeping track of counter variable -> 
`for num in range(1, 101)`